meson-32-cross.txt

```diff
@@ -0,0 +1,17 @@
+[binaries]
+c = 'gcc'
+cpp = 'g++'
+ar = 'ar'
+strip = 'strip'
```

Comment:
NB: This isn't always required, but meson in some cases refuses to build if `[host_machine]` is set while `[binaries]` are not. Though, packaged builds probably aren't going to use this cross-file.